cmd/loadtest/user_listen_test_plan.go

```diff
@@ -227,6 +230,14 @@ func (tp *UserListenTestPlan) registerLaunchFail() {
 	}
 }
 
+func (tp *UserListenTestPlan) registerIncoming() {
+	tp.activityChannel <- l.Activity{
+		Status:  l.StatusIncoming,
+		ID:      tp.id,
+		Message: "Failed launch",
```

Comment:
Chris can you change this message here, cause if I understand correctly its not failing to launch, right?